superbench/benchmarks/result.py

```diff
@@ -64,9 +65,22 @@ def add_raw_data(self, metric, value):
             )
             return False
 
-        if metric not in self.__raw_data:
-            self.__raw_data[metric] = list()
-        self.__raw_data[metric].append(value)
+        export_raw_data = 1
+        try:
+            export_raw_data = int(os.getenv('EXPORT_RAW_DATA', '1'))
```

Comment:
maybe add a prefix "SB_BENCHMARK_" for the env?

Reply:
OK.

Reply:
Agree!

Reply:
Use parameter to replace env variable.

Reply:
It has been removed